tests/unittests/config/test_cc_wireguard.py

```diff
@@ -217,29 +202,28 @@ def test_maybe_install_wg_packages_happy_path(
         distro.install_packages.assert_called_once_with(packages)
 
     @mock.patch("%s.maybe_install_wireguard_packages" % MPATH)
-    def test_handle_no_config(self, m_maybe_install_wireguard_packages):
+    def test_handle_no_config(
+        self, m_maybe_install_wireguard_packages, caplog
+    ):
         """When no wireguard configuration is provided, nothing happens."""
         cfg = {}
         cc_wireguard.handle("wg", cfg=cfg, cloud=None, args=None)
-        self.assertIn(
-            "DEBUG: Skipping module named wg, no 'wireguard'"
-            " configuration found",
-            self.logs.getvalue(),
-        )
-        self.assertEqual(m_maybe_install_wireguard_packages.call_count, 0)
+        assert (
+            mock.ANY,
```

Comment:
Nice approach here. I think I'll take that and use it elsewhere next time I need log level verification :)